test/server/cards/04-MM/Munchling.spec.js

```diff
@@ -0,0 +1,70 @@
+describe('munchling', function () {
+    integration(function () {
+        describe('Munchling\'s fight ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    player1: {
+                        house: 'logos',
+                        archives: ['snufflegator'],
+                        hand: ['eyegor'],
+                        inPlay: ['munchling'],
+                        amber: 0
+                    },
+                    player2: {
+                        inPlay: ['helper-bot']
+                    }
+                });
+                this.player1.fightWith(this.munchling, this.helperBot);
+            });
+
+            it('give prompt', function () {
+                expect(this.player1).toHavePrompt('Any reactions to Helper Bot leaving play?');
```

Comment:
Generally you shouldn't test for this specific test, better to test for a card being clickable or something like that